downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReleaseTarball.java

```diff
@@ -0,0 +1,90 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.DirectoryFileFilter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+public class CreateReleaseTarball {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(String releaseNumber) throws IOException, InterruptedException {
+		logger.info("Running CreateReleaseTarball");
+		String tarDirRelease = "reactome_tar/" + releaseNumber + "/Release";
+		String absReleaseDir = "/usr/local/gkb/scripts/release/website_files_update";
+		String absReactomeDir = "/usr/local/reactomes/Reactome/production/Website/static/download/" + releaseNumber;
+		releaseNumber = releaseNumber.replace("/", "");
+		//TODO: When the dust has settled, make sure the tarball contains everything we need
+		
+		// Remove any existing Release repositorys
+		File releaseDir = new File(tarDirRelease);
+		releaseDir.mkdirs();
+		if (releaseDir.list().length > 0) {
+			Process removeReleaseRepo = Runtime.getRuntime().exec("rm -r " + tarDirRelease);
+			removeReleaseRepo.waitFor();
```

Comment:
Should the directory be removed too or just the files?  Could `FileUtils.cleanDirectory(releaseDir)` be used?

Reply:
Yea, the releaseTarball steps are currently a bit ugly and filled with system calls. I'm going to redo it since the Java system utilities work slightly differently than I was used to and they were giving me a headache :P

Reply:
Sounds good :+1: